src/components/EventsAdminView/EventAdminView.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import { render, screen } from '@testing-library/react'
+import EventAdminView, { EventsAdminViewProps } from './EventAdminView'
+
+describe('Description', () => {
+})
+
+const renderComponent = (props: EventsAdminViewProps) =>
+  render(<EventAdminView  {...props}/>)
```

Comment:
if you dont want to use brackets lets go for this notation, the other is a bit misleading
```suggestion
const renderComponent = (props: EventsAdminViewProps) => render(<EventAdminView  {...props}/>)
```